### ISSUE TITLE
Deriv Bot XML

### ISSUE THREAD

**Issue by jasonwza** [jasonwza/Deriv-Bot#1]
Hello, can anyone help to develop Deriv XML Bot:

Synthetic Indices: Volatility 100 
Rise/Fall
Duration 5 Ticks

Option: Rise

**Trade Conditions:**
There are 5 conditions:

1. The bot will look at the last digits
2. The analysis will only take place when the “worm head” is green.
3. The Colors of numbers.
4. The analysis will be done in 4 numbers.
5. If all conditions are met, the bot will take a UP trade for 5 ticks. 

I will provide the details for the conditions in a PDF file if anyone wishes to help me develop this bot.

**Comment by tyromuwaarits** [comment]
hello can i help you?

**Comment by jasonwza** [comment]
Hi. I have created this pdf which contains all the details and conditions for the Deriv bot. There is actual logic behind the numbers mentioned in the pdf, however, the logic is too difficult to explain. Let me attach the pdf with the details. 
[Deriv DBot.pdf](https://github.com/user-attachments/files/17255545/Deriv.DBot.pdf)
Let me know if you can understand what I have outlined. This specific bot is for experimenting.

**Comment by jasonwza** [comment]
I have another strategy with variable results, but it has much simpler logic than the one I have sent. I'll prepare the pdf for that one and will send that later.

**Comment by tyromuwaarits** [comment]
So what you mean by your explanation is that I just create a bot with the rules that have been explained and enter the 10 patterns into the bot?

**Comment by jasonwza** [comment]
Yes. The bot must see if any of  4 number pattern appears. 
The bot takes an up-trade if all the numbers and their colors match. All numbers must appear consecutively.
Example:

2R 3R 4B 3R.

So if all the numbers appear one after the other, and the logic of the colors is applied, the bot initiates UP trade on 3R. 

R is Red
B is Blue.

Red color in terms of logic means that the current price was below the previous price.
Blue (or Greenish after new deriv update), means that the current price was above the previous price.

**Comment by jasonwza** [comment]
Also the numbers are the last digit of the spot price. Ex: 124. 008, where the last digit is 8.

**Comment by tyromuwaarits** [comment]
Have you tried it manually with this pattern, and if so, what is the accuracy percentage?

**Comment by jasonwza** [comment]
Yes, I have done it manually. It is about 85 to 90% win rate. But analyzing numbers manually is difficult because you only get 2 seconds per number to figure out the trend. And analysis is done with 4 numbers, so a total of 8 seconds.

**Comment by jasonwza** [comment]
The bot is still experimental, and there are hundreds of number combinations. Hence, I will have to add more numbers and test.

**Comment by tyromuwaarits** [comment]
only rise 5 tick right?

**Comment by jasonwza** [comment]
Yes

**Comment by Hitbox6** [comment]
I would like to give the bot a try on my prop farm account

**Comment by jasonwza** [comment]
> I would like to give the bot a try on my prop farm account

You can once the bot is completed.

**Comment by Hitbox6** [comment]
Sure I would love to have that opportunity to be apart of your experiment

**Comment by bayanhadid** [comment]
Hello bro. Ive just found this out. Do you still need help. And how far with the strategy.
